src/components/topics/TopicFactorsList.vue

```diff
@@ -35,61 +35,61 @@ defineProps({
   }
 })
 
-const emits = defineEmits(['updateDatasets'])
+const emits = defineEmits(['updateFactors'])
 
-const modal: Ref<DatasetEditModalType | null> = ref(null)
+const modal: Ref<FactorEditModalType | null> = ref(null)
 const datasetsContent = ref(new Map<string, DatasetV2>())
 
 const { pageConf } = useCurrentPageConf()
 
 const {
-  groupedDatasets,
-  getDatasetIndex,
-  removeDatasetFromGroup,
+  groupedFactors,
+  getFactorIndex,
+  removeFactorFromGroup,
   renameGroup,
   deleteGroup
-} = useGroups(datasetsProperties)
+} = useGroups(factors)
 
 const {
   isFiltering,
-  filterDatasetsProperties,
-  filteredDatasets,
+  filterFactors,
+  filteredFactors,
   isAllGroupsHidden,
   isGroupOnlyHidden
-} = useDatasetFilter(datasetsProperties)
+} = useFactorsFilter(factors)
 
-const { groupedDatasets: filteredResults } = useGroups(filteredDatasets)
+const { groupedFactors: filteredResults } = useGroups(filteredFactors)
 
-const handleRemoveDataset = (group: string, index: number) => {
-  datasetsProperties.value = removeDatasetFromGroup(group, index)
-  emits('updateDatasets')
+const handleRemoveFactor = (group: string, index: number) => {
+  factors.value = removeFactorFromGroup(group, index)
+  emits('updateFactors')
 }
 
 const handleRenameGroup = (oldGroupName: string, newGroupName: string) => {
-  datasetsProperties.value = renameGroup(oldGroupName, newGroupName)
-  emits('updateDatasets')
+  factors.value = renameGroup(oldGroupName, newGroupName)
+  emits('updateFactors')
 }
 
 const handleDeleteGroup = (groupName: string) => {
-  datasetsProperties.value = deleteGroup(groupName)
-  emits('updateDatasets')
+  factors.value = deleteGroup(groupName)
+  emits('updateFactors')
 }
 
 const loadDatasetsContent = () => {
-  datasetsProperties.value.forEach((datasetItem) => {
-    const id = datasetItem.id ?? null
-    if (id && !datasetsContent.value.has(id) && !datasetItem.remoteDeleted) {
+  factors.value.forEach((element) => {
+    const id = element.element?.id ?? null
+    if (id && !datasetsContent.value.has(id) && !element.remoteDeleted) {
       useDatasetStore()
         .load(id, { toasted: false })
         .then((d) => {
           if (d) {
             datasetsContent.value.set(id, d)
-            datasetItem.remoteArchived = !!d.archived
+            element.remoteArchived = !!d.archived
           }
         })
         .catch((err) => {
           if (isNotFoundError(err)) {
-            datasetItem.remoteDeleted = true
+            element.remoteDeleted = true
```

Comment:
```suggestion
  factors.value.forEach((factor) => {
    const id = factor.element?.id ?? null
    if (id && !datasetsContent.value.has(id) && !factor.remoteDeleted) {
     useDatasetStore()
       .load(id, { toasted: false })
       .then((d) => {
         if (d) {
           datasetsContent.value.set(id, d)
            factor.remoteArchived = !!d.archived
         }
       })
       .catch((err) => {
         if (isNotFoundError(err)) {
            factor.remoteDeleted = true
```
Here we're guaranteed to only have factor elements, so s/element/factor/